CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 1.1.0
+
+## ✨ Minor features and improvements
+* Improve line rendering performance by using a more compact line attributes layout ([#8306](https://github.com/mapbox/mapbox-gl-js/pull/8306))
+* Improve data-driven symbol layers rendering performance ([#8295](https://github.com/mapbox/mapbox-gl-js/pull/8295))
+* Add the ability to disable validation during `queryRenderedFeatures` and `querySourceFeatures` calls, as a performance optimization ([#8211](https://github.com/mapbox/mapbox-gl-js/pull/8211)) (h/t [gorshkov-leonid](https://github.com/gorshkov-leonid))
+* Improve `setFilter` performance by caching keys in `groupByLayout` routine ([#8122](https://github.com/mapbox/mapbox-gl-js/pull/8122)) (h/t [vallendm](https://github.com/vallendm))
+* Improve rendering of symbol layers with `symbol-z-order: viewport-y`, when icons are allowed to overlap but not text  ([#8180](https://github.com/mapbox/mapbox-gl-js/pull/8180))
```

Comment:
This changelog entry is true but here's a suggestion if we want to be a little more precise: 

Changed placement order of symbol layers to match the viewport-y order when `symbol-z-order` is set to `viewport-y`, improving collision detection and rendering of symbol layers when icons are allowed to overlap but not text.

Reply:
If the current version is OK I'd stick with that to avoid the message being too verbose, and let people who want to know more details click through to the PR - sound OK?

Reply:
yep!